pwa/app/lib/eventUtils.ts

```diff
@@ -88,3 +90,31 @@ export function getEventWeekString(event: Event) {
       return `Week ${event.week + 1}`;
   }
 }
+
+export function getCurrentWeekEvents(events: Event[]) {
+  const now = new Date();
+  //const now = new Date('Jul 17, 2024');
+  const filteredEvents = [];
+
+  const diffFromWeekStart = now.getDay();
+  const closestStartMonday = new Date(now).setDate(
+    now.getDate() - diffFromWeekStart,
+  );
+
+  for (const event of events) {
+    const startDate = parseDate(event.start_date);
```

Comment:
i think `new Date(event.start_date).getTime()` should just work?
Also maybe rename to `startDateMs` for clarity.